src/components/TransactionStatusToast/StatusIcon.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import styles from './transaction-status-toast.scss';
+import globalStyles from 'assets/sass/main.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInfo, faTimes, faWarning } from '@fortawesome/free-solid-svg-icons';
+import React from 'react';
+import { StatusIconType } from './types';
+
+type StatusIconProps = {
+  type: StatusIconType;
+};
+
+export const StatusIcon: React.FC<StatusIconProps> = ({ type }) => {
+  switch (type) {
+    case 'info':
```

Comment:
you can use the `StatusIconType` instead
also, maybe rename the `StatusIconType` to something that includes Enum so that it is more intuitive